chargeOclock/Source/Other/NetworkAPI.swift

```diff
@@ -47,14 +57,24 @@ class NetworkAPI {
 		performRequest(to: uri, method: "DELETE", completion: completion)
 	}
 
-	private func performRequest(to uri: String, method httpMethod: String, completion: @escaping NetworkCompletion) {
+	private func performRequest(to uri: String, method httpMethod: String, header: [String : String]? = nil, body : [String : String]? = nil, completion: @escaping NetworkCompletion) {
```

Comment:
header => headers
at this point, I'd rather see that body is a data object 🤔  or not, your call